packages/amplify-cli/API.md

```diff
@@ -61,24 +68,18 @@ export class UsageData implements IUsageData {
     getFlowReport(): IFlowReport;
     // (undocumented)
     getSessionUuid(): string;
-    // Warning: (ae-forgotten-export) The symbol "UsageDataPayload" needs to be exported by the entry point index.d.ts
-    //
     // (undocumented)
-    getUsageDataPayload(error: Error | null, state: string): UsageDataPayload;
+    getUsageDataPayload(error: Error | null, state: string): IUsageDataPayload;
     // (undocumented)
-    init(installationUuid: string, version: string, input: CommandLineInput, accountId: string, projectSettings: ProjectSettings, processStartTimeStamp: number): void;
+    init(installationUuid: string, version: string, input: CLIInput, accountId: string, projectSettings: ProjectSettings, processStartTimeStamp: number): void;
```

Comment:
does this init initialize CLI or just metrics collector? if the latter it might use better naming.

Reply:
Gotcha. I was tricked to believe this is global export.